test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1839,6 +1839,14 @@ private void CreateAndValidateScmpFile(SchemaCompareEndpointInfo sourceInfo, Sch
                 ExcludedTargetObjects = null,
             };
 
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["Allow drop blocking assemblies"].Value = true;
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["Drop constraints not in source"].Value = true;
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["Ignore ANSI nulls"].Value = true;
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["No alter statements to change Clr types"].Value = false;
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["Populate files on File groups"].Value = false;
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["Verify deployment"].Value = false;
+            schemaCompareParams.DeploymentOptions.OptionsMapTable["Disable indexes for data phase"].Value = false;
```

Comment:
what is setting these options testing?

Reply:
in line 1826, these values are changed to randomly and verifying buy comparing with the actual default options. to make them in consistent, I have added these values! Thanks